packages/vincent-tools/vincent-tool-sdk/src/lit-action-builder/plugins/polyfillNode.ts

```diff
@@ -0,0 +1,67 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
+import { polyfillNode } from 'esbuild-plugin-polyfill-node';
+
+export const basePluginConfig = {
```

Comment:
Our current esbuild-plugin-polyfill-node configuration is wrong - it uses incorrectly cased strings an leaves a ton of stuff enabled that must be explicitly disabled